javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -123,10 +131,6 @@ public static ResettableClassFileTransformer installBytebuddyAgent(
               .with(new TransformLoggingListener());
     }
 
-    for (AgentBuilder.Listener listener : listeners) {
```

Comment:
Is this not needed anymore?

Reply:
this was only used for tests, now there's TestAgentListenerAccess